Dockerfile.erb

```diff
@@ -14,8 +14,10 @@ RUN apk add --no-cache coreutils curl bash openssl
 
 # Install the necessary plugins
 RUN apk add --no-cache --update --virtual .build-deps \
-      build-base ruby-dev \
-      && gem install fluent-plugin-logdna \
+      build-base ruby-dev git \
+      && git clone -b aptible https://github.com/almathew/fluent-plugin-logdna.git \
+      && gem build fluent-plugin-logdna/fluent-plugin-logdna.gemspec \
+      && gem install fluent-plugin-logdna-0.4.0.gem \
```

Comment:
this isn't properly installing - it leaves an empty directory in the ruby gems directory. When I used https://github.com/rdp/specific_install inside the container to install from the repo/branch it worked though 🤷‍♀️

Reply:
I saw the same behavior so I thought I'd test it out ... looks like the failing step is the `gem build`. You cannot build the gem from outside the `fluent-plugin-logdna` directory because the gemspec is generating the filelist dynamically with " `git ls-files -z`.split("\x0")"

We have to `cd` into the dir, build and install the gem, then `cd out and continue 😅